### DISCUSSION TITLE
Is there a way to update to the latest version without having to delete/add the integration in the config ?

### DISCUSSION THREAD

**Discussion by ItsMe00007** [binsentsu/home-assistant-solaredge-modbus#168] [Q&A]
I am one Version down, but want to use the Power Control Option but don't see it in .storage/core.config_entries.
Do I really have do delete/add the integration to go to the latest version ?

**Comment by AlexanderKas** [comment]
I just deleted and added the 'instance' without losing the entities and the history to set the Power Control feature.

**Reply by ItsMe00007** [reply]
I don't know what you mean with instance. For me an instance is the complete HA installation.
Would you please explain the steps to delete/add the "instance".

**Reply by AlexanderKas** [reply]
Most integrations call it 'Integration entries' or 'devices'. I think it is what you call 'delete the integration'. 
I did the same, but I think by using the same integration/instance name I did not lose any settings or history.

**Comment by binsentsu** [comment]
You could also enable it manually on existing configuration by editing file .storage\core.config_entries. Add
"power_control": true in the data section of integration solaredge_modbus. After that a restart of HA is necessary to activate it. Graphical edit flow for existing integration is still on todo list.

**Reply by ItsMe00007** [reply]
Thanks for the Tip. I thought about that way but I did not know the correct syntax.
I already removed and added the integration because I thought that maybe some part of the code would be missing.
